### ISSUE TITLE
[New Metric] dividing amount accuracy

### ISSUE THREAD

**Issue by Programmer-RD-AI** [PyTorchLightning/metrics#563]
## 🚀 Feature

[New Metric] dividing amount accuracy

So this will be used for regression,

So if we have a small preds and y true

preds = [1000,2000,3000,4000,5000]
y_true = [1250,2500,3250,4526,5262]

we are going to round the preds and the y_true
so then

preds = [1,2,3,4,5]
y_true = [1,2,3,4,5]

I think this is like a small accuracy so this will evaluate the models on a small scale.

I have thought of using something like this.

I don't know if this is already implemented but I think if it's not this is really useful.

With best regards,
Ranuga

**Comment by SkafteNicki** [comment]
Hi @Programmer-RD-AI , thanks for opening this issue.
After looking at your implementation of the PR that you opened, the metric that you are proposing are no different from the `accuracy` metric we already have in torchmetrics except for a division by some factor (`dividing_amount` in your PR) together with rounding. It can therefore already be done like this:
```python
from torchmetrics.functional import accuracy
import torch
dividing_amount  = 1000
preds = torch.tensor([1000,2000,3000,4000,5000])
target = torch.tensor([1250,2500,3250,4526,5262])
accuracy((preds / dividing_amount).round().int(), (target / dividing_amount).round().int()) # tensor(0.8000)
```
I am therefore going to close the issue and PR, as the functionality you are proposing is already existing in torchmetrics.
